### DISCUSSION TITLE
Welcome to ecclesiacrm Discussions!

### DISCUSSION THREAD

**Discussion by phili67** [phili67/ecclesiacrm#1784] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vikingkarwur** [comment]
Hi, thanks for creating EcclesiaCRM. I will be migrating my church database application to EcclesiaCRM (Target is due in July 2021).
My question: I like to use CoreUI (Bootstrap Admin Templates) instead of using AdminLTE as admin template; any instructions for doing this if possible? thank you

**Reply by phili67** [reply]
Everything in the CRM is so deeply integrated to AdminLTE, so to move to something else is really a hard work to do.
I move the code from adminLTE 2 to adminLTE 3 in about 8 months ...

**Comment by dleigh** [comment]
Bonjour,
Je viens de découvrir EcclesiaCRM, mais évidement ce projet existe depuis 2017.  Par contre, il se ressemble tellement à ChurchCRM que je voudrais savoir quelle est la connexion avec le projet ChurchCRM et pourquoi choisir EcclesiaCRM par rapport à ChurchCRM ?
Merci !
David
